File: gta-eenzaam.py
import time
import procfreeze

ProcessName = "GTA5.exe"
#ProcessName = "notepad.exe"

try:
  pid = procfreeze.GetProcessIdFromName(ProcessName)
  if pid:

    print("GTA slaapje laten doen", end = "", flush = True)
    procfreeze.SuspendProcess(pid)

    for i in range(1, 10):
      print(".", end = "", flush = True)
      time.sleep(1)

    print(" ")

    print("GTA weer wakker rammelen")
    procfreeze.ResumeProcess(pid)

    print("ALL GOOD RATTY!")
    
  else:
    print("GTA niet actief")
except:
  print("Oeps, iets fout gegaan!")
  time.sleep(3)
  
time.sleep(2)

File: gta-eenzaam/gta-eenzaam.cpp
#include <windows.h>
#include <iostream>
#include <conio.h>
#include "ProcessUtils.h"

int main()
{
	try {
		DWORD pid = getProcessId("GTA5.exe");

		if (pid > 0)
		{
			std::cout << "GTA slaapje laten doen";

			suspend(pid);

			for (int i = 0; i < 10; i++)
			{
				Sleep(1000);
				std::cout << ".";
			}
			std::cout << std::endl;

			std::cout << "GTA weer wakker rammelen" << std::endl;

			resume(pid);

			std::cout << "ALL GOOD RATTY!" << std::endl;
		}
		else
		{
			std::cout << "GTA niet actief" << std::endl;
		}
	}
	catch (...)
	{
		std::cout << "Oeps, iets fout gegaan!" << std::endl;
		Sleep(3000);
	}

	Sleep(3000);
}

